app/.prettierrc.json

```diff
@@ -2,7 +2,7 @@
   "importOrder": [
     "^@components/(.*)$",
     "^@pages/(.*)$",
-    "^@styles/(.*)$",
+    "^@(styles|src|utils)/(.*)$",
```

Comment:
Notes for reviewers: We have `importOrderSeparation` set to true on line 8, which tells prettier to insert a line break between `import` lines that match any of the elements in the `importOrder` array. This groups all `@components` imports together (no line breaks between those lines) and inserts a line break between that group and the next group of `@pages` imports.

This change makes it so that `import`s that start with `@styles` or `@src` or `@utils` are not separated by from one another by line breaks.